duxy/Day9/eje_one.rb

```diff
@@ -0,0 +1,43 @@
+class EjerOne
+
+  def solution(name)
+    alllines = ""
+    score = 0
+    File.open(name, "r") do |f|
+      f.each_line do |line|
+        alllines = alllines + line
+      end
+    end 
+    numofgroups = 0
+    groups_open = 0
+    novalid = 0
```

Comment:
I can see you are using `novalid` as a boolean. Using int flags can be quite tough to read.